src/modules/sign/components/Signin.css

```diff
@@ -13,15 +13,15 @@
 	margin: 0 auto;
 }
 .sign_in_text {
-	font-family: Inter, serif;
+	font-family: sans-serif;
```

Comment:
Can we have this defined for he document so you don't have to define font-family everywhere?

Reply:
I'll do that.